repose-aggregator/components/filters/uri-user-filter/build.gradle

```diff
@@ -1,12 +1,15 @@
 dependencies {
     compile project(":repose-aggregator/core/core-service-api")
     compile project(":repose-aggregator/commons/commons-utilities")
-    compile project(":repose-aggregator/core/core-lib") //todo: remove this once the wrappers have been removed
     compile "org.slf4j:slf4j-api"
+    compile "org.scala-lang:scala-library"
+    compile "com.typesafe.scala-logging:scala-logging-slf4j_2.10"
 
     provided "javax.servlet:javax.servlet-api"
     provided "javax.inject:javax.inject"
 
     testCompile "junit:junit"
+    testCompile "org.scalatest:scalatest_2.10"
     testCompile "org.mockito:mockito-all"
+    testCompile "org.springframework:spring-test"
```

Comment:
Right. It exists in the Maven build currently, so I assumed we actually needed it for something. I suppose if the tests run in Gradle, we should be good.